fury/actor.py

```diff
@@ -2562,7 +2562,7 @@ def sdf(centers, directions=(1, 0, 0), colors=(1, 0, 0), primitives='torus',
     vtkActor
     """
 
-    prims = {'sphere': 1, 'torus': 2, 'ellipsoid': 3}
+    prims = {'sphere': 1, 'torus': 2, 'ellipsoid': 3, 'superellipse': 4}
```

Comment:
To maintain consistency with the other primitives I suggest renaming it as `superellipsoid`.